### DISCUSSION TITLE
Selection passed to setSelection must point at the current document

### DISCUSSION THREAD

**Discussion by novelAide** [ueberdosis/tiptap#2655] [General Questions]
Hello, have you ever met this? Please help me see, thank you。

[tiptap warn]: BREAKING CHANGE: "defaultOptions" is deprecated. Please use "addOptions" instead. Found in extension: "fontColor

There are many [tiptap warn] prompts

![image](https://user-images.githubusercontent.com/90837360/160375120-4bc954f8-60ce-45a9-a3a8-752cadbb909a.png)

An error will also be reported when adding a new input box
Selection passed to setSelection must point at the current document

![image](https://user-images.githubusercontent.com/90837360/160375385-3c1aad2d-eba9-453a-9c71-99656ab3ad5c.png)

Do you know why? Thank you very much

**Comment by philippkuehn** [comment]
About the warning: you have to update the extensions to the latest versions.
About the error: some example code would help.
